app/assets/images/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
```

Comment:
Can we change this to a more specific name? I don't know how this manifest file is used but I would imagine it supports saving websites as apps on the phone.

Reply:
Doh, I did not mean to add that file.  The manifest is for mulitple android sizes and would need the asset pipeline path in there.

Let me clean that up.